client/reader/list-stream/missing.jsx

```diff
@@ -7,7 +7,7 @@ import React from 'react';
  * Internal dependencies
  */
 import EmptyContent from 'components/empty-content';
-import discoverHelper from 'reader/discover/helper';
+import { isEnabled as discoverIsEnabled } from 'reader/discover/helper';
```

Comment:
I might make a second pass and rename this as 'isDiscoverEnabled` in the helper to clean up import syntax